### DISCUSSION TITLE
Naming of new ndarray-related crates (Rust-only ndarray-linalg alternative)

### DISCUSSION THREAD

**Discussion by quietlychris** [rust-ndarray/ndarray#1160] [Maintainers]
Hello,

A member of the Rust-ML group, @YuhanLiin , has put together a pure-Rust alternative to `ndarray-linalg` library which doesn't require linking to a BLAS library, which is currently located at https://github.com/rust-ml/ndarray-linalg-rs . We're currently considering a breaking release of [`linfa`](https://crates.io/crates/linfa) that includes the use of this library, which would also include publishing the repository above onto crates.io . We wanted to reach out to `ndarray` developers regarding any opinions for the naming a library like this (the current name is somewhat arbitrary), in case there were existing naming conventions around crates in this space or suggestions for one that might better fit the library. We are hoping to release at some point in the near future. Thanks for your consideration!

**Comment by YuhanLiin** [comment]
Note that the crate is still far from feature parity with `ndarray-linalg`. Currently only the algorithms needed by `linfa` are planned for the future.
